src/LanguageServer/Protocol/Handler/Extensions/ExtensionRegisterHandler.cs

```diff
@@ -34,6 +34,9 @@ public async Task<ExtensionRegisterResponse> HandleRequestAsync(ExtensionRegiste
         if (handlerNames.ExtensionException is not null)
             context.Logger.LogException(handlerNames.ExtensionException);
```

Comment:
@dibarbet  i'm still logging this here (as well as passing back to Gladstone).  is that ok with you ?